src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -171,6 +162,14 @@ const AppWrapper = (props: { children: ReactNode }) => {
       };
       handleMessages();
     }
+  }, [isReady, authentication.loggedIn, dispatch]);
```

Comment:
Hmm, but this will create all of these handlers again every time we log in and out?

I think we can just create them once when starting up in `initApp` - and not need the `isReady` state variable.

Reply:
Based on your other comment - I think these can just be done where `onKeriaStatusStateChanged` is setup and remove the useEffect and `isReady`

Reply:
I think we should keep `isReady` and remove `authentication.loggedIn` here because they really don't depend on the `loggedIn` status.